tests/records/test_dumpers.py

```diff
@@ -12,86 +12,150 @@
 from invenio_records.dumpers import ElasticsearchDumper
 
 from invenio_rdm_records.records import BibliographicRecord
-from invenio_rdm_records.records.dumpers import EDTFDumperExt
-
-
-@pytest.mark.parametrize("date, expected_start, expected_end", [
-    ("2021-01-01", "2021-01-01", "2021-01-01"),
-    ("2021-01", "2021-01-01", "2021-01-31"),
-    ("2021", "2021-01-01", "2021-12-31"),
-    ("1776", "1776-01-01", "1776-12-31"),
-    ("2021-01/2021-03", "2021-01-01", "2021-03-31")
-])
-def test_esdumper_with_edtfext(app, db, minimal_record,
-                               date, expected_start, expected_end):
+from invenio_rdm_records.records.dumpers import EDTFDumperExt, \
+    EDTFListDumperExt
+
+
+@pytest.mark.parametrize(
+    "date, expected_start, expected_end",
```

Comment:
minor: is okay, but for the same reason as above maybe for the future is better not to add change lines for simple styling stuff, mainly to keep history untouched. I am guessing is just a formatter that's configured on-save.

Reply:
oh yes, sorry, that was some **Black** magic in action